MultisigTests/Logic/Models/MethodRegistryTestCase.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MethodRegistryTestCase.swift
+//  MultisigTests
+//
+//  Created by Dmitry Bespalov on 17.06.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import XCTest
+@testable import Multisig
+
+class MethodRegistryTestCase: XCTestCase {
+
+    typealias RawTxData = (label: String, method: String, params: [(name: String, type: String, value: String?)])
+
+    func txData(_ d: RawTxData) -> TransactionData {
+        TransactionData(
+            method: d.method,
+            parameters: d.params.map(TransactionDataParameter.init(name:type:value:))  )
```

Comment:
cool. So will it take $0, $1, $2 for name/type/value?

Reply:
yes 😄